packages/gatsby-codemods/src/transforms/global-graphql-calls.js

```diff
@@ -32,14 +32,18 @@ function addEsmImport(j, root, tag) {
     return // already exists
 
   if (!existingImport.length) {
+    const first = root.find(j.Program).get('body', 0)
```

Comment:
We may not actually _need_ this because there will likely be a `import React from 'react';` or `const React = require('react');` as the first line, correct? If that's the case, I can tweak this a bit and just use `insertAfter`